Libraries/Lists/VirtualizedList.js

```diff
@@ -1111,7 +1111,6 @@ class VirtualizedList extends React.PureComponent<Props, State> {
               !this._hasWarned.nesting &&
               this.context == null
             ) {
-              // TODO (T46547044): use React.warn once 16.9 is sync'd: https://github.com/facebook/react/pull/15170
```

Comment:
Hi @ecreeth, I don't think this is unnecessary. `T46547044` is facebook internal tracking for this task. I think this comment is still relevant.